### ISSUE TITLE
Show commonly crafted items on front page

### ISSUE THREAD

**Issue by ryanb** [Kazetsukai/onetech#8]
The current sorting of objects on the initial page is not very useful. It would be better to show a list of commonly crafted items at the top.

Ideally we would track the number of visits per object and sort it based on frequency, but this isn't possible with static pages. Alternatively the list could be manually curated. The player can search for anything else they need.

**Comment by Kazetsukai** [comment]
Yeah not sure what a good sort for this would be. One possibility is recently added items so people could keep up to date with game changes. That's basically what it used to be. I added in a basic "sort objects by their likelihood to spawn", but didn't do anything further.

I'd like to avoid manual curation, if Jason will be adding 100 items per week as his plan was. One possibility would be a utility function like what you were doing for the tech tree: Food adds utility, number of things that can be made with it adds utility, and so on. Then we could just tweak the utility function as necessary.

**Comment by ryanb** [comment]
What do you think of the current sort order based on complexity?

I was thinking we could use Google Analytics to determine the most commonly accessed objects when doing processing.

**Comment by Kazetsukai** [comment]
Feels like it would be a little complex to manage credentials for Analytics etc, and needs an extra step of actually recording when users go to items (haven't got that set up yet, it just records everyone as going to the root url).

I think complexity itself is a bit of a hint, but perhaps things like food value and heat value should be added as ranking things higher. Specifically, we want to rank higher the things that are valuable in gameplay terms, as those are what people should be coming to the site to find out how to make. Stuff like walls is trickier since I don't think there is a straightforward way to distinguish them as being "gameplay relevant".
